dts/bindings/sensor/st,lps22hh-common.yaml

```
@@ -18,9 +18,9 @@ properties:
       default: 0
       description: |
           Specify the default output data rate expressed in samples per second (Hz).
-          Default is power-down mode
+          Default is power-up configuration.
```

```suggestion
          The default of 0 is the power-on reset value.
```

You're saying 0 is PoR value, right? I think this would be a little clearer, because it avoids saying "the default, power down, is the power up configuration", which is kind of confusing :)

---

ok, I can be clearer!                                                   
("It is false to negate that 0 is not the default value" :D )

---

@mbolivar-nordic is it ok now?